game/view/gameplay/actionhud/controlhint.lua

```diff
@@ -0,0 +1,70 @@
+
+local FONT    = require 'view.helpers.font'
+local RES     = require 'resources'
+local PROFILE = require 'infra.profile'
+local COLORS  = require 'domain.definitions.colors'
+
+local Class   = require "steaming.extra_libs.hump.class"
+local ELEMENT = require "steaming.classes.primitives.element"
+local vec2    = require 'cpml' .vec2
+
+local ControlHint = Class{
+  __includes = { ELEMENT }
+}
+
+function ControlHint:init(x, y, texture_name, hints)
+    ELEMENT.init(self)
+    self:setSubtype("control_hints")
+
+    self.pos = vec2(x, y)
+    self.hints = hints
+    self.mode = "DEFAULT"
+
+    self.show = PROFILE.getTutorial("finished_tutorial")
+    self.alpha = 0
+    self.show_speed = 5
+    self.image = RES.loadTexture(texture_name)
+    self.image:setFilter("linear")
+    self.text_font = FONT.get("Text", 20)
+    self.text_font2 = FONT.get("Text", 16)
```

Comment:
actually this seems a recurring problem: magic values. they're bad practice, so i recommend to try not to use arbitrary values like these -- the code very easily becomes unmaintanable

Reply:
in fact, if these values are never changed, why are they even part of the instance itself, and not stored somewhere else?

Reply:
You mean, there should be constant (or, at least, load-time) font objects that this kind of code relies on?

Reply:
Opened issue #1460 from what I reckon you propose.